### ISSUE TITLE
Accessibility issue project creation

### ISSUE THREAD

**Issue by cimigree** [digidem/comapeo-mobile#1506]
**Describe the bug**
A participant was unable to create a new project as their phone system fonts were set to very large sizes. This caused layout overflow that hid or made crucial UI elements inaccessible. 

**To Reproduce**
1. Increase your screen zoom and/ or your font size
2. Go to create a project or rename a default project
3. You will not be able to get to the Name/ Create button

**Expected behavior**
You should be able to create a project even with large display and large fonts

**Screenshots**
<img src="https://github.com/user-attachments/assets/6ae70cde-32d0-4c91-803f-a199d230aa26" width="250" />
